catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -135,12 +133,15 @@ public Object executeWithSubject() throws Exception {
     return metacards;
   }
 
-  private List<Metacard> getMetacardsFromCatalog() throws CatalogCommandException {
+  private List<Metacard> getMetacardsFromCatalog() {
+    return Objects.requireNonNull(security.getSystemSubject())
```

Comment:
Nothing. Just makes it more explicit. Rather than someone going "omg why did this blow up as null, it must be a bug!" They see the code and go "oh." Hopefully :)